src/components/ConnectionsListProfiles.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import UserAvatar from "./UserAvatar";
+import { Button } from "antd";
+import { Profile } from "../utilities/types";
+
+enum ListStatus {
+  CLOSED,
+  FOLLOWERS,
+  FOLLOWING,
+}
+
+interface ConnectionsListProfilesProps {
+  listStatus: ListStatus;
+  connectionsList: Profile[];
```

Comment:
is this a list of profiles that I am connected with or is this all of the social graph?

Reply:
just the list that you are either following or are following you